visualization/CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ find_package(catkin REQUIRED COMPONENTS
   roscpp
   rosconsole
   object_recognition_msgs
+  rostest
```

Comment:
Indeed, I couldn't figure out where it was coming from because you removed it [in this commit](https://github.com/davetcoleman/moveit_ros/commit/7d91cfb643e8d413c8ad60822a03d815a0838508), so I was going to do another sync after this PR. However you encouraged me to track it down to [this](https://github.com/ros-planning/moveit_ros/commit/3617f2edc6e6a54f124592f3c5881d9a974c4207), fixed.